### PR TITLE
Update template for Pull Requests

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Refactored existing template by adding comments `<!-- comment -->`. This way, we can keep the template without deleting anything when creating a new PR.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
+Fixes #
 
-Example:
+<!-- PR description. -->
 
-Title: "Sort observation list by date"
-- [x] Refactor ObservationViewModel allow modification of sort order.
-- [x] Sort results when returned from ObservationRepository.
-[Screenshot](image.gif)
+<!-- Checklist or simple bullet list of things done in the PR. -->
```
Comment (gino-m) [inline .github/pull_request_template.md:8]: Can you add back the check list example? Not everyone is familiar with the - [ ] syntax.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
 
-Fixes #000.
+<!-- Add steps to verify bug/feature. --> 
+
+<!-- Attach a screenshot or GIF (optional) to illustrate the proposed change. -->
```
Comment (gino-m) [inline .github/pull_request_template.md:12]: -> Attach or paste in

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,20 +1,14 @@
...
 
 @... PTAL?
```
Comment (shobhitagarwal1612) [inline .github/pull_request_template.md:14]: How do you feel about removing this line from template and instead using the reviewers/assignees?

Comment (gino-m) [reply .github/pull_request_template.md:19]: Special permissions are needed to assign reviewers; using @mentions will allow anyone who creates an issue to bring attention to it.

**Pull request merged**